run.py

```diff
@@ -1,10 +1,43 @@
 from trackbasket_be import create_app
-
+from flask_socketio import SocketIO, send
+from flask_socketio import join_room, leave_room, emit
+from flask import request
+# from flask import Flask, render_template
 # do this once you're ready for production, etc:
 # config_name = os.getenv('APP_SETTINGS')
 config_name = "development"
 
 app = create_app(config_name)
+# app = Flask(__name__)
+socketio = SocketIO(app, cors_allowed_origins='*', logger=True, engineio_logger=True)
+
+@socketio.on('connect')
+def on_connect():
+  print('just connected')
+
+@socketio.on('joinRoom')
+def on_join(data):
+    id = data['id']
+    print('id: ' + data['id'])
+    join_room(id)
```

Comment:
The code of the `on_join` method gets executed when a `joinRoom` event is sent by the front end